packages/async-apollo-provider/test/__tests__/AsyncApolloProvider.test.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { ApolloClient } from 'apollo-client';
+import { ApolloProvider } from 'react-apollo';
+import { createApolloClient } from '@8base/create-apollo-client';
```

Comment:
Why are you importing `@8base/create-apollo-client`? You can just mock `createApolloClient` function, without any imports.